src/System/Wlog/LoggerConfig.hs

```diff
@@ -256,12 +268,14 @@ topLevelParams =
 
 instance FromJSON LoggerConfig where
     parseJSON = withObject "rotation params" $ \o -> do
-        _lcRotation      <-         o .:? "rotation"
-        _lcTermSeverity  <-         o .:? "termSeverity"
-        _lcShowTime      <- Any <$> o .:? "showTime"    .!= False
-        _lcShowTid       <- Any <$> o .:? "showTid"     .!= False
-        _lcFilePrefix    <-         o .:? "filePrefix"
-        _lcTree          <- parseJSON $ Object $ filterObject topLevelParams o
+        _lcRotation        <-                         o .:? "rotation"
+        -- TODO: Support word "All" which will include all severities at once
```

Comment:
Yes, this is really good idea! Unfortunately, it requires advanced knowledges of `aeson` library to implement this feature... I'm not an `aeson` expert so I can propose only this solution:

1. Perform lookup by key `"termSeveritiesOut"` in `o` (this is possible because [`o` is an `Object`, which is just `HashMap`](http://hackage.haskell.org/package/aeson-1.2.3.0/docs/Data-Aeson.html#t:Object)).
2. Pattern match on result of your lookup. If `Nothing` then `_lcTermSeveritiesOut = Nothing`.
3. If `Just value` then pattern match on the [`value` constructor](http://hackage.haskell.org/package/aeson-1.2.3.0/docs/Data-Aeson.html#t:Value). We're interested only in two constructors: `String` and `Array`.
4. If `Text` inside `String` constructor equals to `All` then return `allSeverities` (otherwise `fail`).
5. If `Array` then call [`Vector.mapM`](http://hackage.haskell.org/package/vector-0.12.0.1/docs/Data-Vector.html#v:mapM)` parseJSON` over resulting array.

I agree, this ugly... But I just don't know how to write better :(

By the way, moreover, we can go even further! Once this implemented, we can support more keywords other than `All` for convenience, like `Debug+`, `Info+`, etc. This will be really handy because often you cares about some level of `Severity` and all levels above that level.

Reply:
Okay, this sounds pretty useful. Also `log-warper` would need good documentation to explain how to use this kind of features.